magnum_cluster_api/resources.py

```diff
@@ -378,7 +380,20 @@ def get_object(self) -> objects.OpenStackMachineTemplate:
             },
             "imageUUID": self.node_group.image_id,
         }
-
+        boot_volume_size = get_label_value(
```

Comment:
let's refactor this to get_int_label_value which parses the value into an in (similar to `get_bool_label_value`)